lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -71,8 +76,13 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 	portLayerClient = client.New(t, nil)
 	portLayerServerAddr = portLayerAddr
 
+	log.Info("Creating image store")
+	if err := createImageStore(); err == nil {
```

Comment:
If you move this inside the `for` loop on line 87, then you won't need to do the loop in `createImageStore()`